### ISSUE TITLE
Modified the picture size, but the result is not good

### ISSUE THREAD

**Issue by littlelingna** [GeorgeSeif/Semantic-Segmentation-Suite#58]
The original image size is 960x720 and the parameter is set to --crop_width 480 --crop_height 352, but the training result is only the top left corner of the image.
I changed the code from:
 def load_image(path):
     image = cv2.cvtColor(cv2.imread(path, -1), cv2.COLOR_BGR2RGB)
     return image
to:
def load_image(path):
    image = cv2.imread(path, -1)
    image = cv2.resize(image, (480, 360))
    image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
    return image
However, the training results are very poor. As the epoch increases, the loss increases and the segmentation results are particularly poor.
I hope you can give me some suggestions. Thank you very much.

**Comment by GeorgeSeif** [comment]
Hhmmm the loss increasing sounds peculiar. Which model are you using and how long did you train for? Your learning rate could be too high.

**Comment by littlelingna** [comment]
Thank you very much for replying me!
    I used the FC_DenseNet model and loss started to increase from the 5th epoch. I solved it by crop the image from the middle instead of zooming out the image. However, the field of vision becomes smaller and only one fourth of the original image is trained instead of the entire image.
    The original image size is 960x720. You are displaying 480x360 on the web page which contains all the information of the image instead of being cropped. How did you do it?

**Comment by GeorgeSeif** [comment]
Ah the one being displayed was done on an older version of that dataset, where the images were 480x360.

Also the custom code you wrote may cause problems in your final accuracy. You are basically resizing AND cropping.

Try a lower learning rate!

**Comment by littlelingna** [comment]
OK, thank you very much!

**Comment by GeorgeSeif** [comment]
No problemo!
